mempool/client.go

```diff
@@ -0,0 +1,39 @@
+package mempool
+
+import (
+	"github.com/pkg/errors"
+	mempoolproto "github.com/tendermint/tendermint/proto/tendermint/mempool"
+)
+
+type mempoolClient struct {
+	mp *Mempool
```

Comment:
"Pointers to interfaces are almost never useful." from [this](https://stackoverflow.com/a/44372954)